Update links in README to use editor-next.swagger.io & update reference to CAMARA API Guideline

maxl2287 opened the pull request:
#### What type of PR is this?

Add one of the following kinds:
* subproject management


#### What this PR does / why we need it:
Changes the usage of viewing Swagger-UI via editor-next.swagger.io

Fixes #325

hdamker commented:
The PR is fine for me, but I have not the impression that using the next-editor from Swagger is a general solution. 
a) the link isn't working as expected for me (not showing the YAML in the left panel when loading)
b) the next editor is an alpha version
c) a own hosted version of Swagger UI, maybe [utilizing GitHub pages](https://github.com/peter-evans/swagger-github-pages) or on camaraproject.org might be the better solution

hdamker commented:
> But I can also investigate or we can try to find a general solution in Commonalities.

I might fit better into Tooling, have created https://github.com/camaraproject/tooling/issues/8 as a start.

maxl2287 commented:
> > But I can also investigate or we can try to find a general solution in Commonalities.
> 
> I might fit better into Tooling, have created [camaraproject/tooling#8](https://github.com/camaraproject/tooling/issues/8) as a start. And yes, the problem was observed by others as well.
> 
> With the now merged links it looks for me like this - left panel is not loading the code: <img alt="image" width="1504" src="https://private-user-images.githubusercontent.com/52109189/445505053-d194fe94-f0ab-4f79-b0aa-6d559832091f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDc3Mjk4MzAsIm5iZiI6MTc0NzcyOTUzMCwicGF0aCI6Ii81MjEwOTE4OS80NDU1MDUwNTMtZDE5NGZlOTQtZjBhYi00Zjc5LWIwYWEtNmQ1NTk4MzIwOTFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTIwVDA4MjUzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU1NTM5OWZhNTM0ODEzZTU5OTQxNzc2NWM1NjZhODcyNTNjNzlkNDQ4NzQ1MTQ2OThiYjdlZDg5OTUxYTZjMGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.1WXDtw4Pn7cMPIJmK7_p5dkCVdBcL9TmCu6d_s3TDUU">

Oh ... okay.
That's weird. I'll investigate.

maxl2287 commented:
@hdamker maybe this is an issue on swagger editor, why I changed it to editor-next.
I faced exactly this issue here:

https://github.com/swagger-api/swagger-editor/issues/5484

That's why I changed it to editor-next.
But at the time when I faced this issue, I did not saw someone complaining on their github-issues site.
